_posts/2024-07-16-Lumi_web_introduction.md

```diff
@@ -0,0 +1,219 @@
+---
+title: 'Getting started with Helmi through LUMI web interface'
+date: 2024-06-19
+permalink: _posts/2024-07-16-Lumi_web_introduction/
+header:
+  teaser: /assets/images/helmi-lumi-web-interface/thumbnail.png
+published: true
+author: Huyen Do
+layout: single
+---
+
+> **In this guide:** Learn how to access the Helmi quantum computer through the LUMI web interface.  
+> **System requirements:** Internet access and a code editor.
+
+With the LUMI web interface, you can easily submit your quantum circuits to be executed on Helmi, currently the first quantum computer in Finland. Future upgrades will bring new machines with 20 and 54 qubits, providing even greater computational power. This hybrid environment allows researchers to accelerate their computations quickly and efficiently, whether running on a real QPU or simulating quantum circuits with GPU acceleration.
```

Comment:
I would recommend referencing 50 qubits instead of 54 for now. This is reflective of FiQCI's roadmap as well.